File: Day01/d1.py
with open("input.txt", "r") as inputFile:
    elves = sorted([sum([int(z) for z in x]) for x in [y.strip().split('\n') for y in inputFile.read().strip().split('\n\n')]])
    print(elves[-1])
    print(sum(elves[-3:]))

File: Day01/d1.rs
use std::fs;

fn main() {
    let input = fs::read_to_string("./input.txt").expect("input file not found");
    let mut lines: Vec<i32> = input
        .split("\r\n\r\n")
        .map(|c| c.split("\r\n").map(|l| l.parse::<i32>().unwrap()).sum())
        .collect();
    lines.sort_unstable();
    println!("{}\n{}",  lines.last().unwrap(),   lines.iter().rev().take(3).sum::<i32>());
}
